kolibri/utils/server.py

```diff
@@ -258,9 +258,12 @@ def run_server(port):
     content_files_handler = cherrypy.tools.staticdir.handler(
         section="/", dir=paths.get_content_dir_path()
     )
+
+    url_path_prefix = conf.OPTIONS["Deployment"]["URL_PATH_PREFIX"]
+
     cherrypy.tree.mount(
         content_files_handler,
-        paths.get_content_url(conf.OPTIONS["Deployment"]["URL_PATH_PREFIX"]),
+        "/" + paths.get_content_url(url_path_prefix).lstrip("/"),
```

Comment:
For code consistency, can you replace this line by:
`"/{}".format(paths.get_content_url(url_path_prefix)),`
the lstrip function is not needed though

Reply:
if lstrip is left out, wouldnt that affect the case where there is no prefix, ie "/" ending up with //5/a/5afdfe.mp4

Reply:
the unit test failures point to the fact that the lstrip is needed